application/src/main/java/org/togetherjava/tjbot/features/help/AutoPruneHelperRoutine.java

```diff
@@ -173,4 +174,14 @@ private void warnModsAbout(String message, Guild guild) {
 
         modAuditLogWriter.write("Auto-prune helpers", message, null, Instant.now(), guild);
     }
+
+    private TextChannel getSelectRolesChannel(JDA jda) {
+        Optional<TextChannel> selectRolesChannelOptional = jda.getTextChannels()
+            .stream()
+            .filter(textChannel -> selectYourRolesChannelNamePredicate.test(textChannel.getName()))
+            .findFirst();
+
+        return selectRolesChannelOptional.orElseThrow(() -> new IllegalStateException(
+                "Could not find role selection channel during AutoPruneHelper routine. Make sure the config is setup properly."));
```

Comment:
the channel isnt necessary for the feature to work. pruning is also more of a background thing, it shouldnt throw exceptions for a missing channel.

have it a warn-log instead and fallback to leaving out the channel part of the DM